### PR TITLE
Resolved Issue #33

### PR DESCRIPTION
**Pull request by soulspark666**
Added smooth scrolling view according to issue #33

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,10 +34,10 @@
         <div class="container">
             <h1 class="display-4">Commons Mobile App</h1>
             <a class="app-badge f-droid-badge" href="https://f-droid.org/repository/browse/?fdid=fr.free.nrw.commons">
-                <img src="images/f-droid-badge.png" alt="F-Droid Logo" />
+                <img src="https://raw.githubusercontent.com/commons-app/commons-app.github.io/master/images/f-droid-badge.png" alt="F-Droid Logo" />
```
[the remark this answers is not in the document]
Comment (soulspark666) [reply index.html]: Both are just fine for me. I used codepen.io to add the smooth scrolling effect, so I placed an absolute link there. If you want I can change it back to relative.

File: index.html [hunk 2]
```diff
@@ -161,12 +161,14 @@ <h2>Commons Mobile App</h2>
         </div>
 
         <div class="qr-code-div">
-            <img src="images/commons-app-qr.png"/>
+            <img src="https://raw.githubusercontent.com/commons-app/commons-app.github.io/master/images/commons-app-qr.png"/>
         </div>
     </footer>
 
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+	<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js'></script>
```
[the remark this answers is not in the document]
Comment (soulspark666) [reply index.html]: Cloudflare servers are a bit faster. You can check it out [here](cdnjs.cloudflare.com).

I can update jquery to [3.3.1](https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js) , I just did'nt saw that while adding it in codepen.io.

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (sivaraam) [reply index.html]: Please revert them to relative paths itself.

[the remark this answers is not in the document]
Comment (soulspark666) [reply index.html]: Done!

File: index.html [hunk 3]
```diff
@@ -7,11 +7,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
     <title>Commons Mobile App</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles/style.css">
 </head>
 
 <body>
-    <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
+    <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-primary">
```
Comment (sivaraam) [inline index.html]: I could see the reason why you're using `fixed-top` for the navbar but it seems to be making the text ("Commons Mobile App") within the `<h1>` [contained within](https://github.com/soulspark666/commons-app.github.io/blob/master/index.html#L35) `<div class="jumbotron jumbotron-fluid">` to be very close to the navbar in the desktop view and it even goes under the navbar in mobile view. Can you fix this?

Comment (sivaraam) [reply index.html]: BTW, sorry for the late request of change. Couldn't get my hands on this for a while :-(

Comment (sivaraam) [reply index.html]: I managed to fix this issue in 4ff0f4c801e1e2fdca6425c7e8ddcb9afc4387ca

**Comment by sivaraam** [comment]
Thanks a lot for your contribution @soulspark666! Hope to see you again :-)

**Pull request merged**